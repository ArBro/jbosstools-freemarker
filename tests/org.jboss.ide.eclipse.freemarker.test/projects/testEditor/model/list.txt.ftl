<#assign seq = ["winter", "spring", "summer", "autumn"]>
<#list seq as x>
  ${x_index + 1}. ${x}<#if x_has_next>,</#if>
</#list> 

<#list 1..3 as n>
  <#list 1..3 as m>
    list item #${n}x${m}
  </#list>
</#list>

<#list seq as x>
  ${x}
  <#if x = "spring"><#break></#if>
</#list>

<#assign x>
  <#list 1..3 as n>
    list item #${n}
  </#list>
</#assign>
Number of words: ${x?word_list?size}
${x}

2.3.23 listing directives:
<#list 1..3>
  Items:
  <#items as n>
  	${n}<#sep>,
  </#items>
<#else>
  No items 
</#list>

<#list 1..3 as x>${x}<#sep>, </#list>

<#list 1..3 as x>
  <div>${x}<#sep>,</#sep></div>
</#list>