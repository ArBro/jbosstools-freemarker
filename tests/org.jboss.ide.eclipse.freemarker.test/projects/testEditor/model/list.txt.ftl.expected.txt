  1. winter,
  2. spring,
  3. summer,
  4. autumn

    list item #1x1
    list item #1x2
    list item #1x3
    list item #2x1
    list item #2x2
    list item #2x3
    list item #3x1
    list item #3x2
    list item #3x3

  winter
  spring

Number of words: 9
    list item #1
    list item #2
    list item #3


2.3.23 listing directives:
  Items:
  	1,
  	2,
  	3
1, 2, 3

  <div>1,</div>
  <div>2,</div>
  <div>3</div>
